pkg/scheduler/factory/factory_test.go

```diff
@@ -310,7 +256,27 @@ func TestDefaultErrorFunc(t *testing.T) {
 		if !exists {
 			continue
 		}
-		handler.ValidateRequest(t, schedulertesting.Test.ResourcePath(string(v1.ResourcePods), "bar", "foo"), "GET", nil)
+		requestReceived := false
+		actions := client.Actions()
+		for _, a := range actions {
```

Comment:
Thanks for updating this. It is easier to read (to me) than the reactor stuff. I almost wonder if this test should assert `len(actions)==1` since the test fails otherwise.